<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
  <widget class="QMainWindow" name="MainWindow">
    <property name="geometry">
      <rect>
      <x>0</x>
      <y>0</y>
      <width>500</width>
      <height>806</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>损的专属OCR</string>
    </property>
    <!-- <widget class="QLineEdit" name="lineEdit">
      <property name="text">
        <string>Lorem ipsum dolor sit amet</string>
      </property>
    </widget> -->
    <widget class="QWidget" name="centralwidget">
      <layout class="QGridLayout" name="gridLayout_1">
        <item row="0" column="1">
          <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="2">
              <widget class="QPushButton" name="pushButton_file_dialog">
                <property name="text">
                <string>导入图片</string>
                </property>
                <!-- <property name="flat">
                <bool>true</bool>
                </property> -->
                <property name="class" stdset="0">
                <string>导入图片</string>
                </property>
              </widget>
            </item>
            <item row="0" column="3">
              <widget class="QPushButton" name="pushButton_2">
                <property name="text">
                <string>识别</string>
                </property>
                <!-- <property name="flat">
                <bool>true</bool>
                </property> -->
                <property name="class" stdset="0">
                <string>识别</string>
                </property>
              </widget>
            </item>
            <item row="0" column="1">
              <widget class="QLineEdit" name="lineEdit_1">
                <property name="enabled">
                  <bool>false</bool>
                </property>
                <property name="placeholderText">
                  <string>Placeholder text</string>
                </property>
              </widget>
            </item>
          </layout>
        </item>
        
        <item row="1" column="1">
          <widget class="QTextEdit" name="textEdit">
            <property name="markdown">
              <string></string>
            </property>
          </widget>
        </item>
        <!-- <item row="2" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
            <widget class="QPushButton" name="pushButton_file_dialog">
              <property name="text">
              <string>QFileDialog</string>
              </property>
            </widget>
            </item>
          </layout>
        </item> -->
    </layout>
    </widget>
    
  </widget>
 <resources/>
 <connections/>
</ui>